jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST

```diff
@@ -11,9 +11,11 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalpost" -c "base ocnanalpost"
 ##############################################
 
 export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ocean}
```

Comment:
Is there a case in which we would want to use a COMOUT that isn't ```${ROTDIR}/${CDUMP}.${PDY}/${cyc}/ocean```?

Reply:
Are we intereacting with other components (eg waves)?

Reply:
no, just ice.

Reply:
... well, so the answer should have been, yes, we're interacting with ice.